include/keylogger/keylogger.h

```diff
@@ -20,7 +20,7 @@ class keylogger
     void handleKeyStroke(DWORD virtualKeyCode, keyStatus status);
 
 
-private:
+public:
```

Comment:
i think this change was for debugging purposes? perhaps switch it back to private?